### DISCUSSION TITLE
failed to upload photos to NocoDB [Nov 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4373] [Discord Threads]
> 🙋‍♂️ **LauraSummer**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### mind to share project info?  
#  
> 🙋‍♂️ **wkw**  
#### are you connecting to any storage plugin?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1035390060910088263)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
